cypress/integration/resmokeLogs/resmoke_logView.ts

```diff
@@ -30,6 +30,61 @@ describe("Basic resmoke log view", () => {
   });
 });
 
+describe("Resmoke syntax highlighting", () => {
+  const logLink =
+    "/resmoke/7e208050e166b1a9025c817b67eee48d/test/1716e11b4f8a4541c5e2faf70affbfab";
+  before(() => {
+    cy.login();
+    cy.visit(logLink);
+    cy.setCookie("has-opened-drawer", "true");
+  });
+  it("should not color non resmoke log lines", () => {
+    cy.dataCy("log-row-0").within(() => {
+      cy.dataCy("resmoke-row").should("have.css", "color", "rgb(0, 0, 0)");
```

Comment:
What if we stored these colors in a variable at the top of the file? like `const colorMap = { black: "rgb(0, 0, 0)"... }` and then test it against that

I think it would make it more obvious what colors these represent, and easier to change in the future